extensions/todoist/package.json

```diff
@@ -183,12 +191,20 @@
       "title": "Todoist Token",
       "description": "Your Todoist Token. Find it in Todoist integration settings view.",
       "placeholder": "Enter your Todoist token"
+    },
+    {
+      "name": "focusFeatureWidth",
+      "type": "textfield",
+      "required": false,
+      "default": "40",
+      "title": "Max width of Focused Task",
+      "description": "Maximum width on the text in the Menu Bar."
     }
```

Comment:
You can put that one in the preferences of the Menu Bar command as it's not global to the whole Todoist extension.